AVFoundation-Combine/Video Player/VideoPlayerViewController.swift

```diff
@@ -65,9 +72,51 @@ final class VideoPlayerViewController: UIViewController {
         videoPlayerContentOverlay.progressSlider.addTarget(self, action: #selector(onSliderThumbTouchedUp), for: .touchUpInside)
     }
     
-    // MARK: UI Actions
+    private func setupReplayControls() {
+        replayOverlay.translatesAutoresizingMaskIntoConstraints = false
+        replayOverlay.backgroundColor = UIColor.black.withAlphaComponent(0.5)
+        view.addSubview(replayOverlay)
+        
+        replayButton = UIButton(type: .custom)
+        replayButton.tintColor = .white
+        replayButton.setImage(UIImage(named: "Play"), for: .normal)
+        replayButton.translatesAutoresizingMaskIntoConstraints = false
+        replayOverlay.addSubview(replayButton)
+        
+        replayButton.addTarget(self, action: #selector(replay), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            replayButton.centerXAnchor.constraint(equalTo: replayOverlay.centerXAnchor),
+            replayButton.centerYAnchor.constraint(equalTo: replayOverlay.centerYAnchor),
+            replayButton.widthAnchor.constraint(equalToConstant: 44),
+            replayButton.heightAnchor.constraint(equalToConstant: 44)
+        ])
+        
+        avPlayerViewController
```

Comment:
I think that you bumped into the same issue i did on `VideoPlayerContentOverlay`when setting up constraints. Although this approach works, it would be preferable if this layout code was written consistently with the other custom UI